impl/src/main/java/org/glassfish/json/JsonArrayBuilderImpl.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2012, 2017 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2012, 2017, 2019 Oracle and/or its affiliates. All rights reserved.
```

Comment:
this should say '2012, 2019', not '2012, 2017, 2019'

Reply:
Fixed.
I wasn't sure how to update the copyright notice. Is there a developer guide somewhere?

Reply:
Yes, there is: https://www.eclipse.org/projects/handbook/#ip-copyright-headers